index.html

```diff
@@ -164,7 +168,7 @@
          */
         function filterByName(searchForName, personList) {
             return personList.filter((person) => {
-                return person.firstName === searchForName || person.lastName === searchForName;
+                return person.fullName.toLowerCase().includes(searchForName.toLowerCase());
```

Comment:
Match full name ("Jon Snow") not just ("Jon") || ("Snow")
transform both fullname property and search text to lowercase. "Jon Snow" != "jon snow"

.include extends the functionality: ("john snow").includes("sno") == true
This is important since the function executes onChange.